build/prefix-source-map.jsm

```diff
@@ -17,4 +17,5 @@
 
 this.EXPORTED_SYMBOLS = [ "SourceMapConsumer", "SourceMapGenerator", "SourceNode" ];
 
+Components.utils.import("resource://gre/modules/devtools/Console.jsm");
```

Comment:
noticed that, but I thought we were ok leaving those in, it will only ever be imported once and it's useful for future debugging?